### ISSUE TITLE
Support raw Kubernetes manifests

### ISSUE THREAD

**Issue by ktarplee** [nikhilsbhat/helm-images#44]
I know this is a helm plugin, but it would be nice to be able to extract images from a directory or file or STDIN of raw Kubernetes manifests (not a helm chart).  Consider this use case.  I have a `skaffold.yaml` that describes how to install a half dozen helm charts and k8s manifests.  I would like to extract all the images in use by the charts in use by the skaffold file, including the values that are added to the charts.  `skaffold render` dumps the raw k8s manifests as YAML to STDOUT.  I would like to be able run `skaffold render | helm images get -raw=- ` and have it dump the images to STDOUT.

So in short I proposing that a flag is added called `-raw` that can take a directory, file, or - (for STDIN) and extract the images from there.  Otherwise I have to make a fake/temporary helm chart to feed into `helm images get` with these manifests.  I suspect `helm template` is effectively called within `helm images get` so we just need to expose that.

**Comment by nikhilsbhat** [comment]
@ktarplee, that sounds interesting! However, I don't have any hands-on experience with `skaffold` yet. Let me read up on it before diving into possible solutions.
